### PR TITLE
Make sure keyboard closing deals with the native context

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ commands.getWindowRect = async function () {
 };
 
 commands.hideKeyboard = async function (strategy, ...possibleKeys) {
-  if ((this.opts.deviceName || '').indexOf('iPhone') === -1) {
+  if (!(this.opts.deviceName || '').includes('iPhone')) {
```
Comment (mykola-mokhnach) [inline lib/commands/general.js]: is that already handled in the other PR?

File: lib/commands/element.js [hunk 2]
```diff
@@ -12,22 +12,26 @@ let commands = {}, extensions = {};
...
+commands.getNativeAttribute = async function getNativeAttribute (attribute, el) {
+  if (attribute === "contentSize") {
+    // don't proxy requests for the content size of a scrollable element
+    return await this.getContentSize(el);
+  }
+
+  // otherwise let WDA handle attribute requests
+  let value = await this.proxyCommand(`/element/${el}/attribute/${attribute}`, 'GET');
+  // Transform the result for the case when WDA returns an integer representation for a boolean value
+  if ([0, 1].indexOf(value) !== -1) {
```
Comment (mykola-mokhnach) [inline lib/commands/element.js]: includes?

File: lib/commands/element.js [hunk 3]
```diff
@@ -12,22 +12,26 @@ let commands = {}, extensions = {};
 // then override anything we want below
 Object.assign(extensions, iosCommands.element);
 
+commands.getNativeAttribute = async function getNativeAttribute (attribute, el) {
+  if (attribute === "contentSize") {
```
Comment (mykola-mokhnach) [inline lib/commands/element.js]: single quotes?

File: lib/commands/general.js (hunk 1 above)
Comment (imurchie) [reply lib/commands/general.js]: Yes. This was done before I did that.

File: lib/commands/element.js (hunk 2 above)
Comment (imurchie) [reply lib/commands/element.js]: Should also be in the other PR.

File: lib/commands/element.js (hunk 3 above)
Comment (imurchie) [reply lib/commands/element.js]: Sure. I'll fix when I rebase with the other changes from #830

**Pull request merged**